api/helpers/CloudHelpers/aws.js

```diff
@@ -0,0 +1,121 @@
+const uuidv1 = require('uuid/v1');
+const AWS = require('aws-sdk');
+
+module.exports = {
+  createBlockBlobFromText_AWS
+};
+
+// Returns [file:BlobResult, fileUrl:string]
+async function createBlockBlobFromText_AWS(
+  containerName,
+  fileName,
+  file,
+  prefix = ''
+) {
+  const ts = Math.round(new Date().getTime() / 1000);
+  const uuidSuffix = uuidv1()
+    .split('-')
+    .pop();
+  const finalName = `${prefix}_${ts}_${uuidSuffix}_${fileName
+    .toLowerCase()
+    .trim()}`;
+
+  const { buffer, mimetype } = file;
+
+  const options = {};
+  if (mimetype && mimetype.length) {
+    options.contentSettings = {
+      contentType: mimetype
+    };
+  }
+
+  /*
+    if you want to use AWS S3 bucket:
+    you have to have "export AWS_ACCESS_KEY_ID=???"
+    and "export AWS_SECRET_ACCESS_KEY=???"
+    variables in your environment variables
+    so that it could login to it
+  */
+  const s3 = new AWS.S3({
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    accessSecretKey: process.env.AWS_SECRET_ACCESS_KEY
```

Comment:
Drop these values from initialization. They will be either picked up by the library automagically or permissions will be granted through the task role in AWS.